web/viewer.css

```diff
@@ -1419,6 +1419,7 @@ html[dir='rtl'] .outlineItemToggler::before {
 .dialog .buttonRow {
   text-align: center;
   vertical-align: middle;
+  clear: both;
```

Comment:
I suppose my question was more whether the `clear: both;` rule is actually necessary at all, now that you're using the new class.